keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -170,33 +172,59 @@ interface and participates in Kubelet pod admission. When the `Admit()`
 function is called, the Topology Manager collects topology hints from other
 Kubelet components.
 
+The Topology Manager calculates the best hint with the following algorithm:
```

Comment:
This should be updated to reflect the latest changes in [81586](https://github.com/kubernetes/kubernetes/pull/81586)